tests/clar_libgit2.c

```diff
@@ -158,6 +158,18 @@ int cl_rename(const char *source, const char *dest)
 
 #endif
 
+int cl_setenv_printf(const char *name, const char *fmt, ...)
+{
+	va_list args;
+	git_buf buf = GIT_BUF_INIT;
+
+	va_start(args, fmt);
+	cl_git_pass(git_buf_vprintf(&buf, fmt, args));
+	va_end(args);
+
+	return cl_setenv(name, git_buf_cstr(&buf));
```

Comment:
OK, I'll move it to the test. I'd put it here in case any other test found it useful, but it seems easy enough for a future test to move the function here if they want to use it.